### PR TITLE
fix(config): add metadata to Ecolink DWZWAVE25

### PR DESCRIPTION
**Pull request by aderusha**
fix(config): add metadata to Ecolink DWZWAVE25

### CODE REVIEW

File: packages/config/config/devices/0x014a/dwzwave25.json [hunk 1]
```diff
@@ -56,6 +56,13 @@
...
+		"exclusion": "1. Start by placing the controller into exclusion mode.\n2. Remove the cover of the sensor, remove the battery for 10 seconds, reinsert the battery, and replace the sensor cover.",
+		"reset": "To restore this sensor to factory default settings, follow the instructions to exclude this sensor from the Z-Wave network. Upon completion of removal from the network the sensor will restore itself to factory default settings automatically.",
+		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=MarketCertificationFiles/1498/D.W.%20Zwave%20Manual%20R1.04.docx"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014a/dwzwave25.json:67]: wow... docx? Do they really not have a PDF? 🙈

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please move the metadata section below the compat section. I prefer to have metadata last.

File: packages/config/config/devices/0x014a/dwzwave25.json (hunk 1 above)
Comment (aderusha) [reply packages/config/config/devices/0x014a/dwzwave25.json:67]: Alright, thanks for the guidance on the metadata!  New commit moves that to the end.

Any guidance on that (admittedly godawful) docx link?  The vendor has no official doc hosted, there are PDFs scattered about the internet but none that are "canonical" in the way that a z-wavealliance.org link probably is.

Comment (AlCalzone) [reply packages/config/config/devices/0x014a/dwzwave25.json:67]: Its fine, we can leave the link as is - after all its from the "official" source. I just didn't expect that any sane manufacturer would provide manuals as something else than PDF.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**